Core/Core/Planner/CalendarEvent/ViewModel/EditCalendarEventViewModel.swift

```diff
@@ -108,6 +115,8 @@ final class EditCalendarEventViewModel: ObservableObject {
     private var selectedCalendar = CurrentValueSubject<CDCalendarFilterEntry?, Never>(nil)
     /// Returns true if any of the fields had been modified once by the user. It doesn't compare values.
     private var isFieldsTouched: Bool = false
+    private let wasEventPartOfSeries: Bool
```

Comment:
Why not `isEventPartOfSeries`?

Reply:
I tired to indicate (along with it being a `let`) that this is about the initial state of the event.
The present state may be different (ie.: user changes to "Does not repeat")

Reply:
Will this differentiation seems overkill to me. 😄
It is just the use of present tense is more common in code. If you were to reason about the use of this in a given context, it is very unlikely that you'd worry about how that was set in the past, rather what would be the value of it at present. In code, it could be a bad thing if a piece of it might have a wide implications outside it.

Reply:
The point is, that we should not use the present value of `isPartOfSeries`.
Renaming to `isInitialEventPartOfSeries` for more clarity.